src/main/java/frc/robot/RobotContainer.java

```diff
@@ -288,75 +356,45 @@ private void configureTestButtonBindsing() {
   }
 
   private void configureButtonBindings() {
+
     SmartDashboard.putData(
         Commands.runOnce(
             () -> {
               SignalLogger.stop();
             }));
 
-    coralScorer
-        .scorerAlignedTrigger()
-        .and(coralScorer.hasCoralTrigger())
-        .and(elevator.elevatorAtSetpoint(ElevatorConstants.kL0Height).negate())
-        .and(elevator.elevatorAtCurrentSetpoint())
-        .whileTrue(
-            Commands.runEnd(
-                () -> {
-                  OI.Driver.setRumble(0.5);
-                  OI.Operator.setRumble(0.5);
-                },
-                () -> {
-                  OI.Driver.setRumble(0);
-                  OI.Operator.setRumble(0);
-                }));
+    if (coralScorer != null && elevator != null) {
+      coralScorer
+          .scorerAlignedTrigger()
+          .and(coralScorer.hasCoralTrigger())
+          .and(elevator.elevatorAtSetpoint(ElevatorConstants.kL0Height).negate())
+          .and(elevator.elevatorAtCurrentSetpoint())
+          .whileTrue(
+              Commands.runEnd(
+                  () -> {
+                    OI.Driver.setRumble(0.5);
+                    OI.Operator.setRumble(0.5);
+                  },
+                  () -> {
+                    OI.Driver.setRumble(0);
+                    OI.Operator.setRumble(0);
+                  }));
+    }
 
     // Elevator Buttons
-    OI.getButton(OI.Driver.A).onTrue(elevator.L0());
-    OI.getButton(OI.Driver.B).onTrue(elevator.L2());
-    OI.getButton(OI.Driver.X).onTrue(elevator.L3());
-    OI.getButton(OI.Driver.Y).onTrue(elevator.L4());
-    OI.getButton(OI.Driver.Start).onTrue(elevator.limitHit());
+    if (elevator != null) {
+      OI.getPOVButton(OI.Driver.DPAD_UP).onTrue(elevator.L0());
+      OI.getPOVButton(OI.Driver.DPAD_LEFT).onTrue(elevator.L2());
+      OI.getPOVButton(OI.Driver.DPAD_RIGHT).onTrue(elevator.L3());
+      OI.getPOVButton(OI.Driver.DPAD_DOWN).onTrue(elevator.L4());
+      OI.getButton(OI.Driver.Start).onTrue(elevator.limitHit());
```

Comment:
Are these the button mapping drive team wants?

Reply:
Elevator has changed to face button now because we wanted to use the POV button for robot orented driving when linning up to reef